src/Nethermind/Nethermind.Consensus/Processing/ReadOnlyTxProcessingEnv.cs

```diff
@@ -48,7 +48,7 @@ public ReadOnlyTxProcessingEnv(
             IWorldState? worldStateToWarmUp = null
             ) : base(worldStateManager, readOnlyBlockTree, specProvider, logManager, worldStateToWarmUp)
         {
-            CodeInfoRepository = new CodeInfoRepository((worldStateToWarmUp as IPreBlockCaches)?.Caches.PrecompileCache);
+            CodeInfoRepository = new CodeInfoRepository(1, (worldStateToWarmUp as IPreBlockCaches)?.Caches.PrecompileCache);
```

Comment:
Why are we passing `1` as `chainId` instead of `specProvider.ChainId`?

Reply:
wrong Pr